src/elements/content-preview/preview-header/PreviewHeader.js

```diff
@@ -70,74 +70,80 @@ const PreviewHeader = ({
                 'bcpr-PreviewHeader--basic': !isPreviewingCurrentVersion,
             })}
         >
-            <FileInfo file={file} version={selectedVersion} />
+            <div className="bp-header bp-base-header">
```

Comment:
Can we add a JSX comment here about this class/element being used by box-annotations? Hopefully that will help avoid this issue from reoccurring.

Reply:
Let's also add a unit test for it so we can make the expectation explicit.